src/Stripe/Infrastructure/Requestor.cs

```diff
@@ -57,8 +83,10 @@ internal static WebRequest GetWebRequest(string url, string method, StripeReques
             if (requestOptions.IdempotencyKey != null)
                 request.Headers.Add("Idempotency-Key", requestOptions.IdempotencyKey);
 
-            request.ContentType = "application/x-www-form-urlencoded";
-            request.UserAgent = "Stripe.net (https://github.com/jaymedavis/stripe.net)";
+            request.Headers.UserAgent.Clear();
+            request.Headers.UserAgent.TryParseAdd($"Stripe.net {Version} (https://github.com/jaymedavis/stripe.net)");
```

Comment:
The content type is not possible to change when the parameters are just in the URL and not in the body of the request. You can only change the content type by selecting specific strongly-typed objects to attach to the HttpRequestMessage.Content property.

Clearing the UserAgent first before trying to set it again is just good practice for working with those headers, since there could be more than one entry in the collection.